usr/src/lib/libsmartsshd/common/mapfile-vers

```diff
@@ -37,11 +38,8 @@
 
 $mapfile_version 2
 
-SYMBOL_VERSION SUNWprivate_1.1 {
+SYMBOL_VERSION SUNWprivate_1.2 {
```

Comment:
This symbol version must not be changed.

Reply:
Damn, good catch.